test/solc/CommandLineInterface.cpp

```diff
@@ -1006,10 +1009,14 @@ BOOST_AUTO_TEST_CASE(cli_include_paths)
 		canonicalWorkDir / "lib",
 	};
 
+	string const expectedStdoutContent = "Compiler run successful, no contracts to compile.\n";
 	OptionsReaderAndMessages result = runCLI(commandLine, "");
 
 	BOOST_TEST(result.stderrContent == "");
-	BOOST_TEST(result.stdoutContent == "");
+	if (SemVerVersion{string(VersionString)}.isPrerelease())
+		BOOST_TEST(result.stdoutContent == "");
+	else
+		BOOST_TEST(result.stdoutContent == expectedStdoutContent);
```

Comment:
This change is fine but this test case is for something else and in the future could change to actually have some output. Could you add a new, simpler case specifically to test for this message?

Reply:
I have added an additional test case for this scenario.